Gemfile.lock

```diff
@@ -129,11 +137,16 @@ GEM
     net-smtp (0.3.3)
       net-protocol
     nio4r (2.5.9)
+feature6/views
+    nokogiri (1.14.4-x64-mingw-ucrt)
+      racc (~> 1.4)
+
```

Comment:
- [ ] Please use `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page
```
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>

```

- [ ] Please your post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)